### ISSUE TITLE
Wrong Psr URLs

### ISSUE THREAD

**Issue by liviubalan** [symfony/symfony-docs#9732]
When referring to [Psr](https://github.com/php-fig) methods a calling like the following is used:

> :method:`Psr\\Cache\\CacheItemPoolInterface::save`

Because Psr is not a Symfony repo this generates a  404 error page.

Example: On https://symfony.com/doc/3.4/components/cache/cache_pools.html#saving-cache-items the "save" URL is broken: http://api.symfony.com/3.4/Psr/Cache/CacheItemPoolInterface.html#method_save

The problem appears on several files: `components/cache/cache_pools.rst`, `components/cache/cache_items.rst`, etc

**Comment by liviubalan** [comment]
Should we use footnotes or should we remove the Psr URLs?

If using footnotes is the right way, how will we bookmark the method pages?
For example: on https://raw.githubusercontent.com/symfony/symfony-docs/3.4/logging.rst we have:
`.. _LoggerInterface: https://github.com/php-fig/log/blob/master/Psr/Log/LoggerInterface.php`

But how will we use this for method `alert` method? If we use https://github.com/php-fig/log/blob/master/Psr/Log/LoggerInterface.php#L43 and the code changes the bookmark will be wrong.

**Comment by javiereguiluz** [comment]
Thinking about this, and precisely because these methods don't belong to `Symfony` namespace, I think it'd be better to show the full namespace + method name instead of linking to the Psr repo:

Before:

```rst
The most common method to save cache items is `save()`_, which stores
the item in the cache immediately (it returns true if the
item was saved or false if some error occurred):
```

After:

```rst
The most common method to save cache items is 
``Psr\\Cache\\CacheItemPoolInterface::save``, which stores
the item in the cache immediately (it returns true if the
item was saved or false if some error occurred):
```

@xabbuh @wouterj what do you think? Thanks!

**Comment by xabbuh** [comment]
I agree with you Javier.

**Comment by javiereguiluz** [comment]
Fixed by #9810.
